packages/babel-helpers/src/helpers/setFunctionName.ts

```diff
@@ -0,0 +1,27 @@
+/* @minVersion 7.23.5 */
+
+// https://tc39.es/ecma262/#sec-setfunctionname
+export default function setFunctionName(
+  fn: Function,
+  name: symbol | string,
+  prefix?: string,
+) {
+  if (typeof name === "symbol") {
+    var description = name.description;
+    name = description ? "[" + description + "]" : "";
+  }
+  name += "";
+  if (prefix) {
+    name = prefix + " " + name;
+  }
+  // In some older browsers .name was non-configurable, here we catch any
+  // errors thrown by defineProperty.
+  try {
+    return Object.defineProperty(fn, "name", {
+      configurable: true,
+      value: name,
+    });
+  } catch (_) {
+    return fn;
+  }
+}
```

Comment:
```suggestion
export default function setFunctionName(
  fn: Function,
  name: symbol | string,
  prefix?: string,
) {
  if (typeof name === "symbol") {
    name = name.description;
    name = name ? "[" + name + "]" : "";
  }
  // In some older browsers .name was non-configurable, here we catch any
  // errors thrown by defineProperty.
  try {
    return Object.defineProperty(fn, "name", {
      configurable: true,
      value: prefix ? prefix + " " + name : name,
    });
  } catch (_) {}
  return fn;
}
```
I'm assuming we always want to return `fn`.

Reply:
Yes, `Object.defineProperty(fn, ...)` will return `fn`. If put `return fn` at the end, we don't have to use the return value of `defineProperty`.

For reference here's TS implementation:
```js
var __setFunctionName = (this && this.__setFunctionName) || function (f, name, prefix) {
    if (typeof name === "symbol") name = name.description ? "[".concat(name.description, "]") : "";
    return Object.defineProperty(f, "name", { configurable: true, value: prefix ? "".concat(prefix, " ", name) : name });
};
```

Reply:
Oops, I didn't notice the first `return` and now I've modified it. :)